lua/entities/hl_camera/projection.lua

```diff
@@ -68,6 +70,11 @@ function ENT:UpdateProjectionRoll(newroll)
 	self.ptexture:SetLocalAngles(Angle(0, 0, newroll))
 end
 
+function ENT:UpdateProjectionOffset(newoffset)
+	if not IsValid(self.ptexture) then return end
+	self.ptexture:SetLocalPos(newoffset)
+end
+
```

Comment:
This part is good, but GitHub isn't letting me comment further down in the file in a section you HAVEN'T changed.

I just playtested this one more time and caught a bug - if you set an offset _and then_ activate the projection, then the projection will be in the wrong place - originating from the camera model instead of the offset.

In ENT:SwitchProjection, there's the line:
```Lua
self.ptexture:SetLocalPos(Vector(0, 0, 0))
```
Please change it to:
```Lua
self.ptexture:SetLocalPos(camera:GetViewOffset)
```

Fingers crossed, this should be the last change I need :)